<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1583</width>
    <height>833</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(193, 223, 195);
font: 9pt &quot;宋体&quot;;</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox_real_1">
    <property name="geometry">
     <rect>
      <x>1290</x>
      <y>10</y>
      <width>221</width>
      <height>171</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(237, 253, 229);
font: 9pt &quot;Arial&quot;;</string>
    </property>
    <property name="title">
     <string>采样</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(246, 255, 255);</string>
     </property>
     <property name="text">
      <string>采样时间：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_sample_time">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>50</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>50</y>
       <width>21</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>ms</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_sample_start">
     <property name="geometry">
      <rect>
       <x>34</x>
       <y>90</y>
       <width>71</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>开始采样</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_sample_stop">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>90</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>结束采样</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_sample_pause">
     <property name="geometry">
      <rect>
       <x>34</x>
       <y>120</y>
       <width>71</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>暂停采样</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_sample_goon">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>120</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>继续采样</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_simulate_1">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>221</width>
       <height>201</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(215, 255, 215);</string>
     </property>
     <property name="title">
      <string>仿真波形</string>
     </property>
     <widget class="QRadioButton" name="radio_wave_sin_6">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>40</y>
        <width>89</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>正弦波</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="radio_wave_square_6">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>80</y>
        <width>89</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>方波</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radio_wave_triangle_6">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>120</y>
        <width>89</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>三角波</string>
      </property>
     </widget>
    </widget>
    <zorder>label_2</zorder>
    <zorder>btn_sample_start</zorder>
    <zorder>btn_sample_stop</zorder>
    <zorder>btn_sample_pause</zorder>
    <zorder>label</zorder>
    <zorder>lineEdit_sample_time</zorder>
    <zorder>btn_sample_goon</zorder>
    <zorder>groupBox_simulate_1</zorder>
   </widget>
   <widget class="QGroupBox" name="groupBox_real_2">
    <property name="geometry">
     <rect>
      <x>1290</x>
      <y>200</y>
      <width>221</width>
      <height>491</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(237, 253, 229);
font: 9pt &quot;Arial&quot;;</string>
    </property>
    <property name="title">
     <string>信号特征</string>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>最大幅值：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>60</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>V</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>最小幅值：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>130</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>V</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>主频率：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>200</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Hz</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>290</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>平均值：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>320</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>V</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_voltages_max">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>60</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-color: rgb(0, 0, 0);
background-color: rgb(247, 255, 224);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_voltages_mean">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>320</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-color: rgb(0, 0, 0);
background-color: rgb(247, 255, 224);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_voltages_Hz">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>200</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-color: rgb(0, 0, 0);
background-color: rgb(247, 255, 224);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_voltages_min">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>130</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-color: rgb(0, 0, 0);
background-color: rgb(247, 255, 224);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_voltages_current">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>380</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-color: rgb(0, 0, 0);
background-color: rgb(247, 255, 224);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>380</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>V</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>350</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>当前值：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_voltages_am">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>260</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-color: rgb(0, 0, 0);
background-color: rgb(247, 255, 224);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>230</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>主频率幅值：</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_simulate_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>231</width>
       <height>491</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(215, 255, 215);</string>
     </property>
     <property name="title">
      <string>信号设置</string>
     </property>
     <widget class="QLabel" name="label_53">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>54</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>波形周期：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_set_period">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>30</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>100</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_54">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>54</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>幅 值 ：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_set_voltages_am">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>70</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>5</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_55">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>54</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>增加直流：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_set_addv">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>110</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_56">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>30</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>ms</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_57">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>80</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>V</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_58">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>120</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>V</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_59">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>54</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>采样周期：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_set_sampletime">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>150</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>10</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_60">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>170</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>ms</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_simulate_start">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>230</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>开始仿真</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_simulate_over">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>230</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>结束仿真</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_simulate_pause">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>270</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>暂停/继续</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_simulate_exit">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>270</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>退出仿真</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_61">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>30</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>ms</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_62">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>70</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>V</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_63">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>110</y>
        <width>21</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>V</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_64">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>160</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>ms</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>340</y>
        <width>54</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>主频率：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_s_freq">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>360</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-color: rgb(0, 0, 0);
background-color: rgb(238, 255, 206);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>360</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Hz</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>440</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>V</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_s_current">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>440</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-color: rgb(0, 0, 0);
background-color: rgb(238, 255, 206);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>420</y>
        <width>54</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>当前值：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_s_am">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>400</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-color: rgb(0, 0, 0);
background-color: rgb(238, 255, 206);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>380</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>主频率幅值：</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>1241</width>
      <height>681</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(56, 113, 0);
font: 14pt &quot;Adobe Devanagari&quot;;
color: rgb(255, 255, 255);
</string>
       </property>
       <property name="title">
        <string>时域波形</string>
       </property>
       <widget class="QWidget" name="widget_time" native="true">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>1221</width>
          <height>301</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(237, 253, 229);</string>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(56, 113, 0);
color: rgb(255, 255, 255);
font: 14pt &quot;Adobe Devanagari&quot;;</string>
       </property>
       <property name="title">
        <string>频域波形</string>
       </property>
       <widget class="QWidget" name="widget_freq" native="true">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>1221</width>
          <height>301</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(237, 253, 229);</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1583</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>导入信号</string>
    </property>
    <addaction name="action_input_txt"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>导出数据</string>
    </property>
    <addaction name="action_time_pic"/>
    <addaction name="action_freq_pic"/>
    <addaction name="action_output_keyvalue"/>
    <addaction name="action_output_voltages"/>
    <addaction name="action_output_freq"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>仿真</string>
    </property>
    <addaction name="action_simulate"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <action name="action_input_txt">
   <property name="text">
    <string>导入txt文件</string>
   </property>
  </action>
  <action name="action_time_pic">
   <property name="text">
    <string>导出时域波形图</string>
   </property>
  </action>
  <action name="action_freq_pic">
   <property name="text">
    <string>导出频域波形图</string>
   </property>
  </action>
  <action name="action_output_keyvalue">
   <property name="text">
    <string>导出信号特征数据</string>
   </property>
  </action>
  <action name="action_wave_sin">
   <property name="text">
    <string>正弦信号</string>
   </property>
  </action>
  <action name="action_wave_square">
   <property name="text">
    <string>方波信号</string>
   </property>
  </action>
  <action name="action_wave_triangle">
   <property name="text">
    <string>三角波信号</string>
   </property>
  </action>
  <action name="action_output_voltages">
   <property name="text">
    <string>导出当前数据</string>
   </property>
  </action>
  <action name="action_output_freq">
   <property name="text">
    <string>导出频率</string>
   </property>
  </action>
  <action name="action_simulate">
   <property name="text">
    <string>启动仿真</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <slots>
  <slot>menu_input_txt_clicked()</slot>
 </slots>
</ui>
